config/settings.yml

```diff
@@ -31,3 +31,7 @@ import:
       directory: museum/records
       only: !ruby/regexp '/met_museum_records.csv/'
       traject_file: met_csv_config
+    numismatics:
+      directory: coins
+      traject_file: numismatic_csv_config
+      agg_provider: Numismatics Society
```

Comment:
Change to "American Numismatic Society" per https://github.com/sul-dlss/dlme/pull/166#discussion_r134897279